plugins/dispatch-webmentions/get-mentions-for-page.js

```diff
@@ -92,10 +94,10 @@ function loadDomFromFile(url, publicDirPath) {
 export default async function getMentionsForPage(url, publicDir, options) {
   const contents = await Promise.all([
     // The old version may have links removed in the new version.
-    options.old ? JSDOM.fromURL(url) : null,
+    options.old ? JSDOM.fromURL(url) : [],
     // The new version may have links not present in the old version.
-    options.new ? loadDomFromFile(url, publicDir) : null
+    options.new ? loadDomFromFile(url, publicDir) : []
   ]);
 
-  return getMentionsFromPages(contents.filter(Boolean), options.ignoredHostnames || []);
+  return getMentionsFromPages(contents.flat(), options.ignoredHostnames || []);
```

Comment:
TS really should understand what I was doing here, but for whatever reason it doesn't. This alternative approaching flat is still JS enough that I'm willing to compromise.